backend/census_historical_migration/sac_general_lib/general_information.py

```diff
@@ -0,0 +1,147 @@
+import audit.validators
+from datetime import date, timedelta
+from census_historical_migration.workbooklib.census_models.census import (
+    CensusGen22 as Gen,
+)
+from census_historical_migration.sac_general_lib.utils import (
+    _census_date_to_datetime,
+)
+from census_historical_migration.base_field_maps import FormFieldMap, FormFieldInDissem
+from census_historical_migration.sac_general_lib.utils import (
+    _create_json_from_db_object,
+    _boolean_field,
+)
+
+mappings = [
+    FormFieldMap(
+        "auditee_fiscal_period_start", "fyenddate", "fy_start_date", None, str
+    ),
+    FormFieldMap("auditee_fiscal_period_end", "fyenddate", "fy_end_date", None, date),
+    FormFieldMap("audit_period_covered", "periodcovered", FormFieldInDissem, None, str),
+    FormFieldMap("audit_type", "audittype", FormFieldInDissem, None, str),
+    FormFieldMap("auditee_address_line_1", "street1", FormFieldInDissem, None, str),
+    FormFieldMap("auditee_city", "city", FormFieldInDissem, None, str),
+    FormFieldMap(
+        "auditee_contact_name", "auditeecontact", FormFieldInDissem, None, str
+    ),
+    FormFieldMap("auditee_contact_title", "auditeetitle", FormFieldInDissem, None, str),
+    FormFieldMap("auditee_email", "auditeeemail", FormFieldInDissem, None, str),
+    FormFieldMap("auditee_name", "auditeename", FormFieldInDissem, None, str),
+    FormFieldMap("auditee_phone", "auditeephone", FormFieldInDissem, None, str),
+    FormFieldMap("auditee_state", "state", FormFieldInDissem, None, str),
+    FormFieldMap("auditee_uei", "uei", FormFieldInDissem, None, str),
+    FormFieldMap("auditee_zip", "zipcode", FormFieldInDissem, None, str),
+    FormFieldMap("auditor_address_line_1", "cpastreet1", FormFieldInDissem, None, str),
+    FormFieldMap("auditor_city", "cpacity", FormFieldInDissem, None, str),
+    FormFieldMap("auditor_contact_name", "cpacontact", FormFieldInDissem, None, str),
+    FormFieldMap("auditor_contact_title", "cpatitle", FormFieldInDissem, None, str),
+    FormFieldMap("auditor_country", "cpacountry", FormFieldInDissem, None, str),
+    FormFieldMap("auditor_ein", "auditor_ein", FormFieldInDissem, None, str),
+    FormFieldMap("auditor_ein_not_an_ssn_attestation", None, None, True, bool),
+    FormFieldMap(
+        "auditor_email", "cpaemail", FormFieldInDissem, "noemailfound@noemail.com", str
+    ),  # FIXME: Do we want to keep `noemailfound@noemail.com`?

```

Comment:
No. We should pick something, though.

`fac-no-email-in-census-data@gsa.gov` or similar.

Whatever we use, if we use it, should be descriptive as to why it is being inserted.

Reply:
I think our process would actually want to treat this as a failure in the first instance? This might be where we want a mapping function.

Reply:
We should treat this as a failure